cucumber.eclipse.editor/src/main/java/cucumber/eclipse/editor/editors/GherkinFormatterUtil.java

```diff
@@ -22,8 +21,14 @@ public static String format(String contents) {
 		// set up
 		StringWriter output = new StringWriter();
 		PrintWriter out = new PrintWriter(output);
-		Formatter formatter = new PrettyFormatter(out, true, false);
+		
+		PrettyFormatter formatter = new PrettyFormatter(out, true, false);
 
+		// TODO: make these configurable preferences!
```

Comment:
:bulb: any help to get these formatter options configurable for the user on a per-project basis would be much appreciated...